fineract-provider/src/main/java/org/apache/fineract/portfolio/loanproduct/api/LoanProductsApiResourceSwagger.java

```diff
@@ -191,7 +191,7 @@ private PostLoanProductsRequest() {}
 
         // Accounting
         @Schema(example = "3")
-        public Integer accountingRule;
+        public Long accountingRule;
```

Comment:
This looks wrong to me based on LoanProduct line 933. It is an ordinal of an enum.

Reply:
Done.